tests/common/ajccValidation.test.js

```diff
@@ -0,0 +1,226 @@
+// Field values are null by default
+const specimen = require('../dummyData/specimen.json');
+const followUp = require('../dummyData/follow_up.json');
+const primaryDiag = require('../dummyData/primary_diagnosis.json');
+
+//use spread operator to grab copies of template files
+const unitTests = {
+    'Specimen' : [
+        [
+            'AJCC Value with <t,n,m>_category values not set.',
+            false,
+            {
+                'row' : {...specimen,
+                    "pathological_tumour_staging_system" : "AJCC 8th Edition"
+                },
+                'name' : 'pathological_tumour_staging_system'
+            }
+        ],
+        [
+            'AJCC Value with one category value not set.',
+            false,
+            {
+                'row' : {...specimen,
+                    "pathological_tumour_staging_system" : "AJCC 6th Edition",
+                    "pathological_m_category": "Some M Value",
+                    "pathological_n_category": "Some N Value",
+                },
+                'name' : 'pathological_tumour_staging_system'
+            }
+        ],
+        [
+            'AJCC Value with two categories value not set.',
+            false,
+            {
+                'row' : {...specimen,
+                    "pathological_tumour_staging_system" : "AJCC 6th Edition",
+                    "pathological_t_category": " ",
+                    "pathological_m_category": "",
+                    "pathological_n_category": "Some N Value"
+                },
+                'name' : 'pathological_tumour_staging_system'
+            }
+        ],
+        [
+            'Non AJCC Value with no categories set.',
+            true,
+            {
+                'row' : {...specimen,
+                    "pathological_tumour_staging_system" : "Aajccc fake edition",
+                },
+                'name' : 'pathological_tumour_staging_system'
+            }
+        ],
+        [
+            'Non AJCC Value with all categories set.',
+            true,
+            {
+                'row' : {...specimen,
+                    "pathological_tumour_staging_system" : "binet",
+                    "pathological_t_category": "some val",
+                    "pathological_m_category": "some val",
+                    "pathological_n_category": "some val"
+                },
+                'name' : 'pathological_tumour_staging_system'
+            }
+        ],
+    ],
+    'FollowUp' : [
+        [
+            'Post-Therapy: AJCC Value with <t,n,m>_category values not set.',
+            false,
+            {
+                'row' : {...followUp,
+                    "posttherapy_tumour_staging_system" : "AJCC 4th Edition"
+                },
+                'name' : 'posttherapy_tumour_staging_system'
+            }
+        ],
+        [
+            'Post-Therapy: AJCC Value with t category value not set.',
+            false,
```

Comment:
should we validate the messages are correct too?

Reply:
messages for this script were custom generated, with many permutations, so that's why I didn't validate the messages 🙈. 

If you think it's worthwhile to add, I can add them. I also didn't want to bloat the files, so let me know your thoughts 👍

Reply:
it's a good starting point we can visit later based on feedback from users